src/System.Security.AccessControl/tests/CommonSecurityDescriptor/CommonSecurityDescriptor_GetSddlForm.cs

```diff
@@ -29,7 +29,7 @@ public static IEnumerable<object[]> CommonSecurityDescriptor_GetSddlForm_TestDat
            yield return new object[] { true, false, 20, "BA", "BG", "64:2:4096:BA:false:0", "0:1:4096:BO:false:0", true , true , false, true , "O:BAG:BGD:(D;;0x1000;;;BO)" };
            yield return new object[] { true, false, 20, "BA", "BG", "64:2:4096:BA:false:0", "0:1:4096:BO:false:0", true , true , true , false, "O:BAG:BGS:(AU;SA;0x1000;;;BA)" };
            yield return new object[] { true, false, 20, "BA", "BG", "64:2:4096:BA:false:0", "0:1:4096:BO:false:0", true , true , true , true , "O:BAG:BGD:(D;;0x1000;;;BO)S:(AU;SA;0x1000;;;BA)" };
-           yield return new object[] { true, false, 0 , null, null, null                  , null                 , true , true , true , true , null};
+           yield return new object[] { true, false, 0 , null, null, null                  , null                 , true , true , true , true , "" };
```

Comment:
> Does netfx agree?

Yup! I verified it against desktop. Not sure how they got set to nulls in the first place.